File: generator/src/java/darkness/generator/scripts/uka17/RGBTest.java
package darkness.generator.scripts.uka17;

import darkness.generator.api.BulbGroup;

public class RGBTest extends BaseScript {
	@Override
	public void run() {
		super.run();
		for (BulbGroup letter : letters) {
			set(letter, 255, 0, 0);
		}
		skip(20);
		for (BulbGroup letter : letters) {
			set(letter, 0, 255, 0);
		}
		skip(20);
		for (BulbGroup letter : letters) {
			set(letter, 0, 0, 255);
		}
		skip(20);
	}
}

File: generator/src/kotlin/darkness/generator/scripts/uka19/RGBTest.kt
package darkness.generator.scripts.uka19

import darkness.generator.api.BulbGroup

class RGBTest: BaseScript() {
    override fun run() {
        super.run()

	for (letter in letters) {
        	set(letter, 255, 0, 0)
	}
	skip(50)
        for (letter in letters) {
		set(letter, 0, 255, 0)
	}
	skip(50)
        for (letter in letters) {
		set(letter, 0, 0, 255)
	}
	skip(50)
    }
}
